### ISSUE TITLE
Error running on OSX

### ISSUE THREAD

**Issue by mnaughto** [hybridgroup/gocv#101]
I'm stumped. I'm trying to get the hello world example running, and I've got everything set up correctly to the best of my knowledge. I've run the `source ./env.sh` script. I've tried both as root and as my regular user. All I get is `signal: killed`, which seems a bit like a C/++ error, and I've tried running `dmesg` like I saw somewhere, but it didn't provide anything useful. 

Any ideas or advice as to how I might investigate this further?

**Comment by deadprogram** [comment]
Hi @mnaughto what macOS version and OpenCV version are you running. Also what version of XCode command line tools?

**Comment by mnaughto** [comment]
```
$ brew info opencv
opencv: stable 3.4.0 (bottled)
```
and

```
$ gcc --version
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 8.1.0 (clang-802.0.42)
Target: x86_64-apple-darwin16.5.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

**Comment by mnaughto** [comment]
And OSX 10.12.4

**Comment by cskonopka** [comment]
@mnaughto I've tried to get this working on OSX with no luck with any version I've been provided by homebrew.

**Comment by deadprogram** [comment]
I think that we have addressed this now in the `dev` branch by using pkg-config on macOS which should pickup the latest version of OpenCV and settings.

I will close this issue on our next release.

**Comment by deadprogram** [comment]
We just released v0.13.0 which includes this code. It should help remove a lot of confusion when getting started with GoCV.

I am going to close this issue now, please reopen if needed. Thank you!
